client/lib/plugins/wporg-data/actions.js

```diff
@@ -54,6 +57,18 @@ var PluginsDataActions = {
 			searchTerm: searchTerm
 		} );
 
+		if ( 'featured' === category ) {
+			this.fetchCuratedList( 'featured', [
+				'woocommerce',
+				'jetpack',
+				'bbpress',
+				'buddypress',
+				'akismet',
+				'vaultpress',
```

Comment:
Probably a bit of overkill unless we need to support multiple lists. If we end up doing the latter, I would probably at least leave a comment that this is hard coded data, and a note on where to find it.